docs/running/wdl.rst

```diff
@@ -0,0 +1,139 @@
+WDL Support in Toil
+********
+
+Support is still in the alpha phase and should be able to handle basic wdl files.  See the specification below for more
+details.
+
+How to Run a WDL file in Toil
+-----------
+Recommended best practice when running wdl files is to first use the Broad's wdltool for syntax validation and generating the
+needed json input file.  Full documentation can be found on the repository_, and a precompiled jar binary can be downloaded here: wdltool_.
+
+That means two steps.  First, make sure your wdl file is valid and devoid of syntax errors by running:
+
+``java -jar wdltool.jar validate example_wdlfile.wdl``
+
+Second, generate a complementary json file if your wdl file needs one.  This json will contain keys for every necessary
+input that your wdl file needs to run:
+
+``java -jar wdltool.jar inputs example_wdlfile.wdl``
+
+When this json template is generated, open the file, and fill in values as necessary by hand.  WDL files all require
+json files to accompany them.  If no variable inputs are needed, a json file containing only '{}' may be required.
+
+Once a wdl file is validated and has an appropriate json file, workflows can be run in toil using:
+
+``toil-wdl-runner example_wdlfile.wdl jsonfile.json``
+
+See options below for more parameters.
+
+ENCODE Example from ENCODE-DCC
+-----------
+To follow this example, you will need docker installed.  The original workflow can be found here:
+https://github.com/ENCODE-DCC/pipeline-container
+
+We've included the wdl file and data files in the toil repository needed to run this example.  First, find the wdl file:
+'src/toil/test/wdl/wdl_templates/testENCODE/encode_mapping_workflow.wdl'
+
+Next, use wdltool_ to validate this file:
+
+``java -jar wdltool.jar validate encode_mapping_workflow.wdl``
+
+Next, use wdltool_ to generate a json file for this wdl file:
+
+``java -jar wdltool.jar inputs encode_mapping_workflow.wdl``
+
+This json file once opened should look like this:
+
+
+    {
+
+    "encode_mapping_workflow.fastqs": "Array[File]",
+
+    "encode_mapping_workflow.trimming_parameter": "String",
+
+    "encode_mapping_workflow.reference": "File"
+
+    }
+
+The trimming_parameter should be set to 'native' and the data files needed for an example run with this workflow are
+found here:
+
+src/toil/test/wdl/ENCODE_data/reference/GRCh38_chr21_bwa.tar.gz
+src/toil/test/wdl/ENCODE_data/ENCFF000VOL_chr21.fq.gz
```

Comment:
A user won't have access to these files unless they have installed toil from source and this isn't explained in the  docs here. Other options are to just link to downloadable copies or just tell them to build from source.

Reply:
Again, I've amended this.  Let me know what you think!